pkg/project/resource_project_repo.go

```diff
@@ -130,7 +131,14 @@ var updateRepos = func(projectKey string, terraformRepoKeys []RepoKey, m interfa
 var addRepo = func(projectKey string, repoKey RepoKey, m interface{}) error {
 	log.Println("[DEBUG] addRepo")
 
-	_, err := m.(*resty.Client).R().
+	_, err := cloneResty(m.(*resty.Client)).
+		SetRetryCount(5).
```

Comment:
The retry count is already 5

Reply:
Initially kept it to 5 so that it can be handled in local testing, now increased to 500.